Pinned messages drawer comments

ibrkhalil opened the pull request:
fixes #16875

status: ready

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 23a4d3b8 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-16998/1/) | 2023-08-12 11:55:12 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-230812-114850-23a4d3-pr16998-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-230812-114850-23a4d3-pr16998-x86.apk)|

ibrkhalil committed:
Pinned drawer design review comments

flexsurfer commented on src/status_im2/contexts/chat/menus/pinned_messages/style.cljs:
```diff
@@ -1,8 +1,9 @@
 
-(def heading
-  {:margin-horizontal 20})
+(defn heading [community]
```
community?

ibrkhalil replied:
Fixed

flexsurfer reviewed (commented):
whats the point of changing icons? looks the same

ibrkhalil committed:
Clean

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

ibrkhalil commented:
One of the design comments was that the icon had a different size than Figma, I thought there was a difference in the file and when git told me they are different I thought sizing is different.

But turned out to be only a color difference 😄

ibrkhalil committed:
Lint

Francesca-G reviewed (approved):
Looks good to me :)

Adding follow up due to the missing community picture in the tag:
<img width="400" alt="Screenshot 2023-08-23 alle 10 23 17" src="https://github.com/status-im/status-mobile/assets/122543089/4255dcc7-869a-4a4b-a0c7-afc1db556713">

design reference:
<img width="400" alt="Screenshot 2023-08-23 alle 10 25 22" src="https://github.com/status-im/status-mobile/assets/122543089/7fbcb156-1e94-46a6-8424-0a3009b7803a">

status-im-auto commented:
## 74% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 11
Passed tests: 32
```
```
IDs of failed tests: 703194,702732,703495,702948,702783,703297,703202,702786,702731,702808,702958 
```
<h3>Failed tests (11)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=16998&TR_CASE_IDS=703194,702732,703495,702948,702783,703297,703202,702786,702731,702808,702958">Rerun failed tests</a></li><br/><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218989">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ActivityCenterElement` by `xpath`: `//*[contains(@text, 'user1')]/ancestor::*[@content-desc='activity']` will be presented </div><div>Device 2: Find `Button` by `accessibility id`: `open-activity-center-button` </div></blockquote></p>

```
medium/test_activity_center.py:315: in test_activity_center_admin_notification_accept_swipe
    self.home_2.clear_chat_long_press(self.username_1)
../views/home_view.py:519: in clear_chat_long_press
    self.get_chat(username).long_press_element()
../views/home_view.py:346: in get_chat
    chat_in_ac.wait_for_element(20)
../views/base_element.py:122: in wait_for_element
    raise TimeoutException(
 Device `2`: `ActivityCenterElement` by` xpath`: `//*[contains(@text, 'user1')]/ancestor::*[@content-desc='activity']` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0da8947b60b04668b395b998e9991b63?auth=628d063345aa8f500f020f615102a646#307">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee269303e7294470a3ff09ca14984c86?auth=4d4d65aced415ce505e6f8a1e8a49d10#148">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218992">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><p><blockquote><div>Device 2: ChatMessageInput element not found </div><div>Device 2: Looking for a message by text: gallery </div></blockquote></p>

```
critical/test_public_chat_browsing.py:679: in test_community_several_images_send_reply
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Gallery message was not received
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#199">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#153">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218987">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div># STEP: Device 1 sends a message with hashtag in the dogs channel</div><div>Device 1: Sending message '#cats' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1195: in test_community_hashtag_links_to_community_channels
    self.channel_1.send_message(message_with_hashtag)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:122: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#711">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#598">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218966">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div># STEP: Invited member gets push notification with the mention and tap it</div><div>Device 2: Getting PN by 'user_2' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1088: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Push notification with the mention was not received by admin
E    Can not edit a message with a mention
E    Push notification with the mention was not received by the invited member 
```

<b>[[Issue with username in PN, issue #6 in https://github.com/status-im/status-mobile/issues/15500]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#530">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#410">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218964">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'test message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1377: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message status was not changed to Delivered, it's Sent after back up online!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#461">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#402">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218958">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218959">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218997">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:353: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:634: in execute_script
    return self.execute(command, {
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:321: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:31: in check_response
    raise wde
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:26: in check_response
    super().check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218994">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:353: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:634: in execute_script
    return self.execute(command, {
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:321: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:31: in check_response
    raise wde
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:26: in check_response
    super().check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
 error processing request. If this error persists, please contact SauceLabs support.
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218993">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
critical/chats/test_group_chat.py:235: in test_group_chat_reactions
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
critical/chats/test_group_chat.py:235: in <listcomp>
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
../views/base_view.py:390: in navigate_back_to_home_view
    self.driver.press_keycode(4)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/extensions/keyboard.py:106: in press_keycode
    self.execute(Command.PRESS_KEYCODE, data)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:321: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:31: in check_response
    raise wde
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:26: in check_response
    super().check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:208: in check_response
    raise exception_class(value)
 The test with session id b12fa455ea95437d9287bb5105880170 has already finished, and can't receive further commands.
E   For help, please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b12fa455ea95437d9287bb5105880170?auth=ef1e077ad0fd128c3a0309c2f797f773#143">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb2ca58f3a914216bb70d409d592bbc9?auth=ad575725e211ab4ba7b0e477facbabf4#98">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7d11a61b06b04cd68f54085df206c695?auth=467563b56b09ca517b9e5eecd2faedf3#78">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218968">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:353: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:634: in execute_script
    return self.execute(command, {
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:321: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:31: in check_response
    raise wde
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:26: in check_response
    super().check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
 session backend not connected
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (32)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218967">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b12fa455ea95437d9287bb5105880170?auth=ef1e077ad0fd128c3a0309c2f797f773#125">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb2ca58f3a914216bb70d409d592bbc9?auth=ad575725e211ab4ba7b0e477facbabf4#71">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7d11a61b06b04cd68f54085df206c695?auth=467563b56b09ca517b9e5eecd2faedf3#71">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218988">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0da8947b60b04668b395b998e9991b63?auth=628d063345aa8f500f020f615102a646#270">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee269303e7294470a3ff09ca14984c86?auth=4d4d65aced415ce505e6f8a1e8a49d10#130">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218985">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0da8947b60b04668b395b998e9991b63?auth=628d063345aa8f500f020f615102a646#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee269303e7294470a3ff09ca14984c86?auth=4d4d65aced415ce505e6f8a1e8a49d10#111">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218986">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0da8947b60b04668b395b998e9991b63?auth=628d063345aa8f500f020f615102a646#155">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee269303e7294470a3ff09ca14984c86?auth=4d4d65aced415ce505e6f8a1e8a49d10#114">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218969">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#585">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#468">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218982">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#220">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#193">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218973">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#267">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#249">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218990">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#381">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218984">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#315">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218972">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218971">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218976">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#288">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#264">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218975">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#154">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#111">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218977">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#708">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#568">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218974">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2558c1dd1edf49ff9ded6160b167d56a?auth=69be5b19b8924ff3f4806ba12dd4d7ff#329">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/13b8852bfaef452db23771da2c3a9ce2?auth=f2434b3893d9a76c3d2d9d418a678279#305">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218991">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#254">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218961">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218999">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218983">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218978">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#80">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218995">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/493d951715f04407abdb34bc11c78114?auth=589e3e551bb9d8a3d2f51de0457f150e#159">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218963">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#130">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218960">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#401">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#333">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218965">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#570">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#578">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218970">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#261">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#197">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218962">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#192">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#149">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218998">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#502">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#565">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218981">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#314">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218996">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#341">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#272">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218957">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7ac7d0ca4a74e2fa719085683600af0?auth=b9cf6a10e707812102db55d4e9fd464b#78">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c80dc029031241c184532ffad667fd74?auth=596a91ce2d4064bea74d4be9206b102b#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218980">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8747238bcbf46f69d4dabdf775ee017?auth=ddf52429270d5c03fa5e6e049c688f28#90">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785969d9719140129a66b94523fff5ad?auth=f6710c6fbe269ac2bc9de0ea8f3d1974#91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3218979">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8747238bcbf46f69d4dabdf775ee017?auth=ddf52429270d5c03fa5e6e049c688f28#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785969d9719140129a66b94523fff5ad?auth=f6710c6fbe269ac2bc9de0ea8f3d1974#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

ibrkhalil committed:
Add icon in drawer

ibrkhalil commented:
Added the community icon on the drawer
![image](https://github.com/status-im/status-mobile/assets/33176106/ccdfef6c-9ff8-46e2-b6df-ddac3135c885)

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

status-im-auto commented:
#### :heavy_check_mark: [status-mobile/prs/ios/PR-16998#9](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-16998/9/) :small_blue_diamond: ~8 min 5 sec :small_blue_diamond: 89d3d75d :small_blue_diamond: [:package: ios package](https://i.diawi.com/Vfpbbe)

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

ibrkhalil committed:
Merge branch 'develop' into design-notes/pinned-messages-drawer

churik commented:
@ibrkhalil 
 did you create an issue follow-up based on the design review?

Based on https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md > Design review, all follow-up issues have to be created when PR is merged and linked to the original PR.

ibrkhalil commented:
> @ibrkhalil did you create an issue follow-up based on the design review?
> 
> Based on https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md > Design review, all follow-up issues have to be created when PR is merged and linked to the original PR.

I addressed the review comments by Francesca in [this](https://github.com/status-im/status-mobile/pull/16998#issuecomment-1694261463) comment and when she approved I took it as it was correctly fixed/addressed.
